prody/proteins/pdbfile.py

```diff
@@ -590,16 +593,34 @@ def _parsePDBLines(atomgroup, lines, split, model, chain, subset,
             resnames[acount] = resname
             chainids[acount] = chid
             if isPDB:
-                try:
-                    resnums[acount] = int(line[22:26])
-                except ValueError:
+                if dec:
+                    try:
+                        resnum = int(line[22:26])
```

Comment:
Please consider assigning `line[22:26]` to a variable above for clarity.

Reply:
Good idea.